gandalf.js

```diff
@@ -1,67 +1,59 @@
-const squaredEvenNumbers = numbers.filter(num => num % 2 === 0).map(num => num ** 2);
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?